### PR TITLE
Fixes & various stuff

### PR DESCRIPTION
**Pull request by robertgzr**
Change-type: patch
Signed-off-by: Robert Günzler <robertg@balena.io>

### CODE REVIEW

File: hostapp.go [hunk 1]
```diff
@@ -9,38 +9,44 @@ import (
...
-	Driver     string `json:"Driver"`
+
+	ID     string `json:"ID"`
+	Image  string `json:"Image"`
+	Name   string `json:"Name"`
+	Driver string `json:"Driver"`
 }
 
 type Container struct {
 	Config
```
Comment (robertgzr) [inline hostapp.go]: is this is a subset of the type [here](https://pkg.go.dev/github.com/docker/docker@v17.12.0-ce-rc1+incompatible/api/types/container?tab=doc#Config) + HostConfig

I wonder if we should call it container, as that's slightly misleading... The `label` argument of [`Mount`](https://github.com/balena-os/hostapp/blob/master/hostapp.go#L166)
also references a label on a container _image_, right?

I mean concept wise it's true so I don't mind really

Comment (robertgzr) [reply hostapp.go]: I just realized that this still needs setup before... the container referenced here still needs to be created by the engine-host daemon right?

**Comment by robertgzr** [comment]
since I can't comment in here without making a change, let's do it like this:

* Is this used anywhere?
https://github.com/balena-os/hostapp/blob/4509339fb7c704ce179b46855067691f0b5382e8/hostapp_test.go#L153

* would be cool to make this env vars, then `env ROOT_DIR=/path/to/rootdir go test -v` would work
https://github.com/balena-os/hostapp/blob/4509339fb7c704ce179b46855067691f0b5382e8/hostapp_test.go#L15-L16

**Comment by robertgzr** [comment]
also I added dockerfile and script to enable ci, that makefile should also work for the meta-balena integration

**Review by alexgg — DISMISSED** [review]

File: hostapp.go (hunk 1 above)
Comment (alexgg) [reply hostapp.go]: hi @robertgzr yes, the container it references need to be created by the engine-host daemon. Would you prefer a different name?

**Pull request closed without merging**